docs/storybook/stories/forms/FormFieldCEP.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Button, Flex } from '@ttoss/ui';
+import { Form, FormFieldCEP, useForm, yup, yupResolver } from '@ttoss/forms';
+import { Meta, Story } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Forms/FormFieldCEP',
+  component: FormFieldCEP,
+} as Meta;
+
+const schema = yup.object({
+  cep: yup.string().required('Value is required'),
+  cepDesabled: yup.string(),
+});
+
+const Template: Story = () => {
+  const formMethods = useForm({
+    mode: 'all',
+    resolver: yupResolver(schema),
+  });
+
+  React.useEffect(() => {
+    formMethods.setError('cep', { message: 'Value is required' });
```

Comment:
Não seria melhor deixar a mensagem de erro em português? Aplica-se para as outras mensagens de erro que estão presentes no componente.

Reply:
ttoss is open-source and its default language is en. Someday we'll improve our i18n to use it always.